docs/estela/api/endpoints.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Enpoints
+parent: API
+grand_parent: estela
+---
+
+# Endpoints
+The project offers different endpoints to interact with the system. For example:
+
+### api/auth/login
```

Comment:
Perhaps another example, more related to the spiders and jobs, should be more descriptive.

Reply:
@rodp63, it's ready to review.